docs/Cloud_integrations/LoRaWAN/LoRagateway.md

```diff
@@ -0,0 +1,101 @@
+# LoRaWAN Gateway
+
+## Overview
+
+A LoRaWAN gateway is a radio module that functions as a communication device between end devices and a LoRaWAN network server (LNS) within a LoRa network.
```

Comment:
Shouldn't we refer to it as `LoRaWAN Gateway`?

Reply:
Also, I think there is no such thing as `LoRa network` - LoRa is only a radio communication technique. The devices form a LoRaWAN network. There is often a problem to distinguish the LoRa and the LoRaWAN.